src/part-3-intro.md

```diff
@@ -1,7 +1,9 @@
-# Part 3: The Compiler Frontend
+# Part 3: Source Code Representations
 
-This part of the guide looks at the compiler frontend (with the major exception
-of the type system, which is in the next part). This part describes the process
-of taking raw source code from the user and transforming it into various forms
-that the compiler can work with easily. These are called intermediate
-representations.
+This part of the guide looks at the various ways the compiler transforms and
+represents source code. This part describes the process of taking raw source
```

Comment:
I think first sentence must be removed... the second one says pretty much the same thing, only better.